### PR TITLE
Update docs.html

### PR DESCRIPTION
**Pull request by hussiantabha**

### CODE REVIEW

File: docs.html [hunk 1]
```diff
@@ -1,452 +1,716 @@
...
-            <img class="avatar-sm" src="https://avatars.dicebear.com/api/avataaars/aliasger.svg"/>
-            <img class="avatar-md" src="https://avatars.dicebear.com/api/avataaars/aliasger.svg" />
-            <img class="avatar-bg" src="https://avatars.dicebear.com/api/avataaars/aliasger.svg" />
+      <div class="logo">
+        <h2>Anti-Strap UI</h2>
+      </div>
+      <div class="nav-link-container">
+        <a href="index.html" class="nav-link">Home</a>
+        <a href="#" class="nav-link">Docs</a>
+        <a href="#" class="nav-link">Github</a>
```
Comment (rkapoor10) [inline docs.html:18]: add the links here

Comment (rkapoor10) [reply docs.html:18]: also GitHub main button on landing page is also not linked

**Review by rkapoor10 — APPROVED** [review]
The code looks good; approved for merge.

**Review by paarthbajaj — APPROVED** [review]

File: app.css [hunk 2]
```diff
@@ -7,229 +7,231 @@
...
-    display: flex;
-    flex-direction: column;
-    justify-content: space-between;
-    padding: 1rem 0;
-    margin-top: 2rem;
-}
-.display-title{
-    margin-bottom: 2.5rem;
+.display-code-content {
+  border: 1px solid #dcdcdc;
```
Comment (paarthbajaj) [inline app.css]: You could use a variable for this grey color

**Review by paarthbajaj — APPROVED** [review]
All looking good. You can merge.

**Pull request merged**